pyvista/ext/__init__.py

```diff
@@ -0,0 +1 @@
+"""Module level init for ``pyvista.ext``."""
```

Comment:
Shouldn't this be either less, or more?

Reply:
Our docs complain about not having summary line, so I added one.  Could add more, but this matches what we have in our other init modules.